nginx/templates/nginx.conf

```diff
@@ -4,7 +4,7 @@
 #   * Official Russian Documentation: http://nginx.org/ru/docs/
 
 user              nginx;
-worker_processes  1;
+worker_processes  2;
```

Comment:
This should come from the pillar so it is configurable - there's a slim chance that this could mess up some projects, and if we are changing it from one value to another we should make it easier in the future and let it be overridden from the pillar/changed per project without having to edit the formula